examples/scripts/script_ps_selection.py

```diff
@@ -0,0 +1,125 @@
+"""Example script for secting PS from SLCs.
+
+This .py script is designed to be executed with a Dask SLURMCluster on a SLURM managed HPC system.
```

Comment:
Hi @FreekvanLeijen, in principle I would like to separate them, since the scripts in this folder are mostly SLURM oriented.

I would like to propose I write 1-2 example Jupyter Notebooks for local (laptop) execution. In general I think when you are executing locally, the easy visualization and interaction are more important. Thus I would like to propose Notebooks as local example.  But I am happy to hear your opinion.